labordata/lm20
# Handle unexpected dates

## Overview
There were parsing errors coming from unexpected inputs in the `termDate` field. Examples of these inputs are `03312022`, `0130/22`, `Not req'd to submit`, `on-going`, or `045/15/21`. This branch:

- handles the first two cases, by formatting them as `month/day/year`
- handles the cases with words, by making them lower case for standardization sake, and passing them through
- notifies the user of the last case, but passes it through as is


## Testing Instructions
- Run make `lm20.db` and check for errors along the way
  - The process runs in github actions and this might be enough to satisfy this test
